src/agent/src/image_rpc.rs

```diff
@@ -26,8 +26,12 @@ use image_rs::image::ImageClient;
 use std::io::Write;
 
 const AA_PATH: &str = "/usr/local/bin/attestation-agent";
-const AA_KEYPROVIDER_PORT: &str = "127.0.0.1:50000";
-const AA_GETRESOURCE_PORT: &str = "127.0.0.1:50001";
+
+const AA_KEYPROVIDER_URI: &str =
+    "unix:///opt/confidential-containers/attestation-agent/keyprovider.sock";
+const AA_GETRESOURCE_URI: &str =
+    "unix:///opt/confidential-containers/attestation-agent/getresource.sock";
```

Comment:
it would make more sense to keep these domain sockets in `/run/...`

Reply:
Yes, you're right! Thanks. We'll fix this from AA side, and then will update this PR

Reply:
Is this hardcoded? Or can we pass the socket path as a command line argument when starting AA

Reply:
This is a default value in attestation-agent, and surely can pass from command line. For kata-agent, using default value is enough I think